### PR TITLE
Redo README

### PR DESCRIPTION
**Pull request by steelbrain**
This updates the description to indicate that the package can work in both Chrome & Node.js

Amazon link taken from here https://github.com/rhysmorgan134/react-carplay/#prerequisites

### CODE REVIEW

**Comment by steelbrain** [comment]
Thoughts @gozmanyoni  @rhysmorgan134?

**Review by rhysmorgan134 — COMMENTED** [review]
Just some thoughts, apologies I put these on a while ago but forgot to submit!

File: README.md [hunk 1]
```diff
@@ -107,11 +93,12 @@ Contributions are what make the open source community such an amazing place to b
 5. Open a Pull Request
 
 
+## License
```
[the remark this answers is not in the document]
Comment (gozmanyoni) [reply README.md:96]: Not sure you can do that with open source (If you forbid any kind of use it wont fit the definition of open source) - But I don't think its something big to worry about - MIT is very popular though and widely used

File: README.md [hunk 2]
```diff
@@ -40,62 +36,52 @@
...
-configuration settings and also downloads the APK file thats usually used with it. The APK file then gets extracted and its contents get sent over usb to the
-dongle itself. The dongle then sends a h264 bytestream from the phone, this contains the video data. And it also sends an audio stream.
+This repository contains the npm package `node-carplay` that can be used on the Web or in Node.js. It allows interfacing with the [Carlinkit USB adapter](https://amzn.to/3X6OaF9) and stream audio/video on your computer. The package can be used in the Node.js environment using native USB bindings ([`libudev-dev` required](https://github.com/node-usb/node-usb#prerequisites)), or in Chrome (or equivalent browsers) using [`WebUSB` API](https://developer.mozilla.org/en-US/docs/Web/API/WebUSB_API).
 
-### Built With
+There are multiple Carplay dongles on the market, the ones that convert wired to wireless carplay WILL NOT WORK. You need one that converts android/factory infotainment systems into Carplay (CPC200-Autokit or CPC200-CCPA etc). The package forwards video feed in h264, and PCM audio coming in from the USB dongle.
```
[the remark this answers is not in the document]
Comment (gozmanyoni) [reply README.md:45]: Yeah - or leveraging the wiki here to add this info

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (steelbrain) [reply README.md:96]: Agreeing with @gozmanyoni here. I think MIT is in good spirits with open source, and given how small this niche probably is, cant expect a lot re commercial revenue.

That aside, even if we make the change, the previous versions of the sources that ship with MIT (which is the current license as per the license file) will continue to be valid to old sources so most likely any commercial consumer could just fork from there and continue

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (steelbrain) [reply README.md:45]: Ideally we’d document related things into the repository itself.

I’ve experienced this a lot and maybe you have too but browsing around some forums, you find links that have expired years ago. So while this is actively maintained now, in the future the circumstances could be different. Github repository will probably outlive self hosted forums

File: README.md [hunk 3]
```diff
@@ -40,62 +36,52 @@
...
+```
 
-<!-- ROADMAP -->
-## Roadmap
+## Usage
 
-* - [ ] Enabled hardware accelleration
-* - [x] Replace jsmpeg with lighter alternative
-* - [x] Make wireless operational
-* - [x] Integrate microphone
```
[the remark this answers is not in the document]
Comment (steelbrain) [reply README.md:95]: That could be one way, we could also have a pinned Github issue for the roadmap. Which one do you think would be better?

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rhysmorgan134) [reply README.md:96]: Ok cool, MIT read the best when I did the initial research, let's stay as is then!

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (rhysmorgan134) [reply README.md:95]: Personally I quite like an overall view on the readme, but I agree each item should have a pinned issue, maybe any agreed features could just be tagged as such in an issue and then readme updated to reflect a consolidated list?

**Pull request merged**